src/pages/resultsView/mutation/ResultsViewMutationMapperStore.ts

```diff
@@ -62,7 +65,10 @@ export default class ResultsViewMutationMapperStore extends MutationMapperStore
         >,
         public uniqueSampleKeyToTumorType: {
             [uniqueSampleKey: string]: string;
-        }
+        },
+        public generateGenomeNexusHgvsgUrl: (hgvsg: string) => string,
+        protected genomenexus_client?: GenomeNexusAPI,
+        protected genomenexus_internal_client?: GenomeNexusAPIInternal
```

Comment:
Let's use camelcase for variable naming:
```suggestion
        protected genomenexusClient?: GenomeNexusAPI,
        protected genomenexusInternalClient?: GenomeNexusAPIInternal
```